internal/privatemessaging/message.go

```diff
@@ -107,7 +99,12 @@ func (s *messageSender) setDefaults() {
 	if s.msg.Header.Type == "" {
 		s.msg.Header.Type = fftypes.MessageTypePrivate
 	}
-	if s.msg.Header.TxType == "" {
+	switch s.msg.Header.TxType {
+	case fftypes.TransactionTypeUnpinned, fftypes.TransactionTypeNone:
+		// "unpinned" used to be called "none" (before we introduced batching + a TX on unppinned sends)
+		s.msg.Header.TxType = fftypes.TransactionTypeUnpinned
+	default:
+		// the only other valid option is "batch_pin"
 		s.msg.Header.TxType = fftypes.TransactionTypeBatchPin
```

Comment:
Should we yell at them for setting an invalid txtype? Or just quietly set to the default like this?

Also might consider adding similar pre-validation to `broadcast/message.go` - can't remember if/when that case actually fails due to an invalid txtype.

Reply:
Going to go for the default approach, but do the same for Broadcast